content/applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst

```diff
@@ -1,101 +1,271 @@
+
+Since the |MPS| allows for manual intervention, it is useful for replenishing products where the
+demand of existing sales orders (SOs) does **not** reflect probable future demand.
+
+.. example::
+   A retail store sells artificial *Christmas trees* during the holiday season. It is currently
+   September, and the store has less than ten Christmas tree |MOs| confirmed for the month of
+   December.
+
+   Despite the number of confirmed |MOs|, the procurement manager knows that the demand for
+   Christmas trees in December is going to be much higher, once the holiday season starts. As a
+   result, they manually enter a greater demand in the |MPS|, so they can properly replenish the
+   product in time for the increase in customer demand.
+
+.. important::
+   It is essential to remember that the |MPS| is a **MANUAL** tool. Adding a product to the |MPS|
+   does not cause it to be manufactured or purchased automatically. The |MPS| simply suggests the
+   amount of the product that should be replenished, but requires user input to create the |MOs| or
+   |POs| that are used to replenish it.
+
+   For this reason, it is recommended that the |MPS| **NOT** be used alongside reordering rules for
+   the same product. Because reordering rules are an automated workflow, they conflict with the
+   manual replenishment method of |MPS|. Using both, in unison, can lead to inaccurate forecasts and
+   the creation of unnecessary replenishment orders.
+
+Enable and configure MPS
+========================
+
+To use the |MPS| feature, navigate to :menuselection:`Manufacturing app --> Configuration -->
+Settings`, and tick the :guilabel:`Master Production Schedule` checkbox in the :guilabel:`Planning`
+section. Finally, click :guilabel:`Save`.
+
+After enabling the :guilabel:`Master Production Schedule` feature, two new fields appear under it on
+the :guilabel:`Settings` page: :guilabel:`Time Range` and :guilabel:`Number of Columns`.
+
+The :guilabel:`Time Range` field is used to select the period of time over which planning takes
+place, and offers three options: :guilabel:`Monthly`, :guilabel:`Weekly`, and :guilabel:`Daily`. For
+example, if :guilabel:`Monthly` is selected, the |MPS| plans the production requirements of products
+and components on a monthly basis.
+
+The :guilabel:`Number of Columns` field is used to specify the quantity of the selected
+:guilabel:`Time Range` units shown on the |MPS| page. For example, if the :guilabel:`Time Range`
+field is set to :guilabel:`Monthly`, and `12` is entered in the :guilabel:`Number of Columns` field,
+the |MPS| shows one column for the next 12 months, starting with the current month.
+
+If the values of the :guilabel:`Time Range` or :guilabel:`Number of Columns` fields are altered,
+click :guilabel:`Save` again to save the changes.
+
+.. image:: use_mps/mps-setting.png
+   :align: center
+   :alt: The MPS setting in the Manufacturing app's settings.
+
+MPS UI
+======
```

Comment:
Suggestion to use "dashboard" instead of the "UI" user interface acronym that may not be familiar to some readers
```suggestion
|MPS| dashboard
===============
```